Ruby/syntax_test_ruby.rb

```diff
@@ -174,7 +174,7 @@ def my_method(param1, param2)
 #                             ^ punctuation.definition.group.end
   end
 
-  def self.my_second_method *arg_without_parens
+  def self.my_second_method *arg_without_parens # comment.line.number-sign.ruby
```

Comment:
Can you add some assertions here to make sure the comment you added is actually a comment? Right now, your additions don't actually check anything.

http://www.sublimetext.com/docs/3/syntax.html#testing has info about how to do that.